ecosystem/python/sdk/aptos_sdk/bcs.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This is a simple BCS serializer and deserializer. Learn more at https://github.com/diem/bcs
+"""
+
+from __future__ import annotations
+import io
+import typing
+import unittest
+
+MAX_U8 = 2**8
+MAX_U16 = 2**16
+MAX_U32 = 2**32
+MAX_U64 = 2**64
+MAX_U128 = 2**128
```

Comment:
These all need to be `-1`.

e.g. If there were `2` bits, `2**2 = 4`, but the maximum value of `2` bits is `3`.

Reply:
Hmm... dammit